src/vconsole/vconsole-setup.c

```diff
@@ -328,48 +320,88 @@ static void setup_remaining_vcs(int fd, bool utf8) {
         }
 }
 
+static int find_source_vc(char **ret_path, int *ret_idx, const char *src_vc) {
+        int ret_fd;
+
+        if (src_vc) {
+                _cleanup_close_ int fd = -1;
+
+                if (!(*ret_path = malloc(strlen(src_vc) + 1))) {
+                        log_oom();
+                        return -1;
```

Comment:
please don't make up errors such as -1. Please only return negative errno style errors, i.e. "return -ENOMEM" or so. Note that log_oom() returns -ENOMEM anywa,y hence you can simplify the above then to:

```
return log_oom();
```

which is a very typical idiom in the systemd sources